build.sh

```diff
@@ -103,6 +104,12 @@ if [ -f "./${TEST_FILENAME}" ]; then
 	echo "=>  Executing tests"
 	# Next command is to workaround the fact that docker-compose does not use .dockercfg to pull images
 	cat ./${TEST_FILENAME} | grep "image:" | awk '{print $2}' | xargs -n1 docker pull
+
+	if [ -z "$IMAGE_NAME" ]; then
```

Comment:
Don't do this - just `unset DOCKERCFG` after dumping it in the file (L46). The test container should never be able to read that file anyway.